src/Resources/config/doctrine/BaseUser.orm.xml

```diff
@@ -10,7 +10,7 @@
         <field name="website" type="string" column="website" length="64" nullable="true"/>
         <field name="biography" type="string" column="biography" length="1000" nullable="true"/>
         <field name="gender" type="string" column="gender" length="1" nullable="true"/>
-        <field name="locale" type="string" column="locale" length="8" nullable="true"/>
+        <field name="locale" type="string" column="locale" length="10" nullable="true"/>
```

Comment:
Isn't this a BC break? I think it should be targeted on master

Reply:
Well, if we just change it here everyone using sonata user would get a database error. You can still do http://docs.doctrine-project.org/projects/doctrine-orm/en/latest/tutorials/override-field-association-mappings-in-subclasses.html. Let's see and wait for others. ping @sonata-project/contributors

Reply:
If we are changing this it would be nice to give it a few extra chars, to not having this problem again. And probably change it on master to avoid bc breaks

Reply:
OK for me. Updated the base branch and increased to 15 the length